ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -62,7 +62,7 @@
     private static final By LOCATION = By.id("w3");
     private static final By WHO_WHEN_WHERE = By.id("who-when-where");
     private static final By REQUEST_APPOINTMENT = By.linkText("Request Appointment");
-    private static final By APPOINTMENT_EDIT = By.cssSelector("div.info-section:nth-child(4) > div > a:nth-child(3)");
+    private static final By APPOINTMENT_EDIT = By.cssSelector("#content > div.dashboard.clear.row > div.col-12.col-lg-9 > div > div:nth-child(1) > div:nth-child(7) > div.info-header > a > i");
```

Comment:
please use more solid element look ups, you can try using element attribute selection such as; `element[attribute="value"]` e.g. `a[target="_blank"]`

Reply:
> please use more solid element look ups, you can try using element attribute selection such as; `element[attribute="value"]` e.g. `a[target="_blank`

ok,let me revisit this @k-joseph

Reply:
please @HerbertYiga check this out and update